### PR TITLE
fix: all module net and text dependent package CVE-2022-41717 and CVE-2022-32149 vulnerabilities

### PR DESCRIPTION
**Pull request by fengshunli**
Signed-off-by: fsl <1171313930@qq.com>

image

### CODE REVIEW

**Comment by KnVerey** [comment]
This repo has a large number of modules, and many others also have indirect dependencies on those two packages. Can you please update all of them?

**Comment by fengshunli** [comment]
> This repo has a large number of modules, and many others also have indirect dependencies on those two packages. Can you please update all of them?

yes, it has been updated

**Comment by KnVerey** [comment]
> 
> This repo has a large number of modules, and many others also have indirect dependencies on those two packages. Can you please update all of them?
> 
> > yes, it has been updated

Perhaps you forgot to push a new commit? I'm still only seeing changes to the one module

**Comment by fengshunli** [comment]
> > This repo has a large number of modules, and many others also have indirect dependencies on those two packages. Can you please update all of them?
> > > yes, it has been updated
> 
> Perhaps you forgot to push a new commit? I'm still only seeing changes to the one module

update

**Comment by KnVerey** [comment]
The latest push still only changes kustomize/go.mod and kustomize/go.sum, not any of the other modules.

**Comment by fengshunli** [comment]
what else do i need to do @KnVerey

**Comment by fengshunli** [comment]
> The latest push still only changes kustomize/go.mod and kustomize/go.sum, not any of the other modules.
ok

File: kustomize/go.mod [hunk 1]
```diff
@@ -8,11 +8,11 @@ require (
 	github.com/spf13/cobra v1.4.0
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.8.0
-	golang.org/x/text v0.3.7
+	golang.org/x/text v0.5.0
 	sigs.k8s.io/kustomize/api v0.12.1
 	sigs.k8s.io/kustomize/cmd/config v0.10.9
 	sigs.k8s.io/kustomize/kyaml v0.13.9
-	sigs.k8s.io/yaml v1.2.0
+	sigs.k8s.io/yaml v1.3.0
```
Comment (KnVerey) [inline kustomize/go.mod:15]: Please remove this unrelated bump

**Review by KnVerey — COMMENTED** [review]
Looks like you've got all the modules now, thanks! Please remove the unrelated bump, and run `make test-go-mod` to pick up a few missed `.sum` updates.

File: kustomize/go.mod (hunk 1 above)
Comment (fengshunli) [reply kustomize/go.mod:15]: updated

**Pull request merged**